home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/exception/ExceptionMessages.java

```diff
@@ -0,0 +1,142 @@
+    public static final String ALERT_WRONG_INVITATION_TYPE_MESSAGE = "Wrong invitation type.";
+
+    public static final String NOT_MATCH_POLL_QUESTION_TYPE_MESSAGE = "Type of the PollQuestion doesn't match";
+
+    public static final String NOT_FOUND_QUESTION_TYPE_MESSAGE = "Type of the PollQuestion is not found";
+
+    public static final String INVALID_HOUSE_MESSAGE = "Can't add or remove house, invalid poll_id or house_id";
+
+    public static final String NOT_FOUND_NEWS_MESSAGE = "Can't find news with given ID:";
+
+    public static final String NOT_FOUND_COOPERATION_FORMAT_MESSAGE = "Can't find cooperation with given ID: %d";
+
+    public static final String NOT_FOUND_CONTACT_FORMAT_MESSAGE = "Can't find contact with given ID:%d";
+
+    public static final String NOT_FOUND_TYPE_CONTACT_FORMAT_MESSAGE = "Type of the contact is not found";
+
+    public static final String NOT_MATCH_TYPE_CONTACT_FORMAT_MESSAGE = "Type of the contact doesn't match";
+
+    public static final String NOT_FOUND_CONTACT_ID_MESSAGE = "Contact with id: %d is not found";
+
+    public static final String ALERT_USER_HAS_MAIN_CONTACT_MESSAGE = "User with id %d already has main %s contact";
+
+    public static final String NOT_FOUND_COOPERATION_MESSAGE = "Cooperation with 'id: %s' is not found";
+
+    public static final String NOT_FOUND_INVITATION_FORMAT = "Invitation with id: %d not found.";
+
+    public static final String NOT_FOUND_REGISTRATION_TOKEN_MESSAGE = "Registration token not found";
+
+    public static final String ALERT_COOPERATION_HAS_MAIN_CONTACT_MESSAGE = "Cooperation with id %d already has "
+        + "main %s contact";
+
+    public static final String ALERT_INVITATION_OVERDUE_MESSAGE = "Invitation was overdue";
+
+    public static final String ALERT_INVITATION_NOT_ACTIVE_MESSAGE = "Invitation is not active";
+
+    public static final String ALERT_INVITATION_DELETED_BY_ADMIN_MESSAGE = "Invitation was deleted "
+        + "By Cooperation Admin";
+
+    public static final String ALERT_INVITATION_ALREADY_EXIST_MESSAGE = "Invitation already exist for cooperation";
+
+    public static final String ALERT_INVITATION_ALREADY_EXIST_APARTMENT_MESSAGE = "Invitation already exist "
+        + "for apartment";
+
+    public static final String ALERT_APARTMENT_WITH_NUMBER_EXIST_MESSAGE = "Apartment with number %s already exist "
+        + "in this house";
+
+    public static final String ALERT_HOUSE_EXIST_IN_POLL_MESSAGE = "House with id:%s already exists in poll with id:%s";
+
+    public static final String NOT_FOUND_APARTMENT_ID_MESSAGE = "Apartment with 'id: %d' is not found";
+
+    public static final String NOT_FOUND_HOUSE_WITH_ID_IN_COOPERATION_MESSAGE = "House with id: %d not found "
+        + "in this cooperation";
+
+    public static final String NOT_FOUND_COOPERATION_WITH_ID_MESSAGE = "Can't find cooperation with given ID: %d";
+
+    public static final String NOT_FOUND_HOUSE_WITH_ID_MESSAGE = "House with 'id: %d' is not found";
+
```

Comment:
Please, remove this one unnecessary empty line. You don't need it before closing bracket